### DISCUSSION TITLE
Can't install VCLibs on Windows 11 Insider Preview

### DISCUSSION THREAD

**Discussion by rodrigoswz** [LSPosed/MagiskOnWSALocal#485] [Q&A]
I get this error whenever I try to run the script.

I tried to install this manually but got an error saying it's not possible as it needs Windows Installer closed.

So I tried it through Powershell, and it says it's not possible because it needs Windows Terminal closed. So I have no idea how to get this installed, since my Windows build has Powershell built into the Windows Terminal only.

![image](https://user-images.githubusercontent.com/14058427/230918190-5f9a47c6-6061-4294-b975-810bba1dc13a.png)
![image](https://user-images.githubusercontent.com/14058427/230918252-635e39ed-d877-433d-9ee1-18b71bddecc0.png)

**Comment by CarlGao4** [comment]
Maybe you can try using the latest commit? Or you can manually download the file and place it in the `MagiskOnWSALocal\download` folder
